cmd/internal/apidump/apidump.go

```diff
@@ -192,40 +189,20 @@ var Cmd = &cobra.Command{
 }
 
 func init() {
-	Cmd.Flags().Var(
-		&outFlag,
-		"out",
-		"The location to store the trace. Can be an AkitaURI or a local directory. Defaults to a trace on the Akita Cloud. Exactly one of --out or --project must be specified.")
-	Cmd.Flags().MarkDeprecated("out", "For use by Akita users")
-
 	Cmd.Flags().StringVar(
-		&serviceFlag,
+		&projectID,
 		"project",
 		"",
-		"Your Akita project. Exactly one of --out or --project must be specified.")
-	Cmd.Flags().MarkDeprecated("project", "For use by Akita users")
+		"Your Postman Insights projectID.")
 
 	Cmd.Flags().StringVar(
 		&postmanCollectionID,
 		"collection",
 		"",
-		"Your Postman collectionID.")
+		"Your Postman collectionID. Exactly one of --project, --collection must be specified.")
```

Comment:
Why are we still supporting `--collection`? One of our stated goals is to have users only reinstall once. Won't those that continue to use `--collection` need to reinstall again once we drop support for this in the back end?